.env.production.sample

```diff
@@ -134,3 +134,6 @@ STREAMING_CLUSTER_NUM=1
 # If you use Docker, you may want to assign UID/GID manually.
 # UID=1000
 # GID=1000
+
+# Maximum allowed character count
+# MAX_CHARS=500
```

Comment:
I'd suggest MAX_TOOT_CHARS or something similar, as one may want custom maximum length for other things (such as bios).

Reply:
Makes sense, i made that change.